### PR TITLE
Feature/7 network page

### PR DESCRIPTION
**Pull request by woutslakhorst**
part of #7 

- Adds client for node peer diagnostics
- Adds web api for passing it to the frontend
- Adds page showing a svg with the network
- PeerIDs as mouse over

Screenshot 2023-04-07 at 10 34 06

### CODE REVIEW

**Review by gerardsn — COMMENTED** [review]
Some of the nodes disappear from view (see image), and some minor textual things / questions.

**Comment by gerardsn** [comment]
The current version of the node appends `-bootstrap` to the peerID it sends in its bootstrap connections. Any peerID containing this postfix should probably be ignored in the network topology?

File: codegen/diagnostics-config.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package: diagnostics
+generate:
+  echo-server: false
+  client: true
+  models: true
+output-options:
+  client-type-name: diagnosticsClient
```
[the remark this answers is not in the document]
Comment (woutslakhorst) [reply codegen/diagnostics-config.yaml:7]: gives you a different struct name, so you don't have 2 names that collide/are confusing.

**Comment by woutslakhorst** [comment]
> The current version of the node appends `-bootstrap` to the peerID it sends in its bootstrap connections. Any peerID containing this postfix should probably be ignored in the network topology?

These are deduplicated in the `realPeerID` func

**Review by gerardsn — APPROVED** [review]

**Pull request merged**